### PR TITLE
Port to popperjs v1

### LINKED ISSUE #284: Upgrade to popper.js v1 when it comes out of beta
Popper.js has a v1 beta out, and we should expect it to release v1 soon. We should keep a close eye on it and validate/upgrade when we're able to.

### PR DESCRIPTION
**Pull request by mttwc**
Fixes #284 

@VishaalK can you please verify this change as well? Can't see anything broken but it would be good to have a second pair of eyes
@FezVrasta

### CODE REVIEW

File: src/scripts/clipperUI/components/popover.tsx [hunk 1]
```diff
@@ -25,19 +25,8 @@ class PopoverClass extends ComponentBase<{}, PopoverProps> {
...
-			if (mainControllerElem) {
-				// We want to set the parent lower in the HTML hierarchy to avoid z-index issues relating to stacking contexts
-				popoverObj.parent = mainControllerElem;
-			}
-
-			this.refToPopper = new popperJS(document.getElementById(this.props.referenceElementId), popoverObj, {
+			let popperElement = this.generatePopperElement(Constants.Ids.mainController);
+			this.refToPopper = new popperJS(document.getElementById(this.props.referenceElementId), popperElement, {
 				placement: this.props.placement,
 				modifiersIgnored: this.props.modifiersIgnored,
```
Comment (FezVrasta) [inline src/scripts/clipperUI/components/popover.tsx]: `modifiersIgnored` is deprecated. Full migration guide at https://github.com/FezVrasta/popper.js/issues/62

Comment (mttwc) [reply src/scripts/clipperUI/components/popover.tsx]: Fixed, cheers.

**Comment by VishaalK** [comment]
Its good to go!

**Comment by mttwc** [comment]
:shipit:

**Pull request merged**